### PR TITLE
Add cookbook entry on mass upload of assets

### PR DESCRIPTION
**Pull request by MarieMinasyan**

### CODE REVIEW

File: cookbook/product_asset/mass_import.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+How to Mass Import Assets
+=========================
+
+A command has been created in the PIM to allow you to mass import assets: ``php app/console pim:product-asset:mass-upload``
+
+For this command to work, you need to copy files you want to import into the folder configured in ``pim_parameters.ym`` under the parameter ``tmp_storage``.
+The command will upload the files into the application.
+
+For example, if you upload a file named "demo_video.wmv", Akeneo PIM will check if an asset with the code "demo_video" exists.
+If yes, the application will update the asset "demo_video" by importing the file "demo_video.wmv" as the new reference file of the asset (and will also generate the variations for the asset).
```
Comment (rybus) [inline cookbook/product_asset/mass_import.rst]: If so

File: cookbook/product_asset/mass_import.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
+How to Mass Import Assets
+=========================
+
+A command has been created in the PIM to allow you to mass import assets: ``php app/console pim:product-asset:mass-upload``
+
+For this command to work, you need to copy files you want to import into the folder configured in ``pim_parameters.ym`` under the parameter ``tmp_storage``.
+The command will upload the files into the application.
+
+For example, if you upload a file named "demo_video.wmv", Akeneo PIM will check if an asset with the code "demo_video" exists.
```
Comment (juliensnz) [inline cookbook/product_asset/mass_import.rst]: wmv is pretty dead an not open, maybe a open source format ? like ogg for example ? It's a details, not a big deal

File: cookbook/product_asset/mass_import.rst (hunk 1 above)
Comment (rybus) [inline cookbook/product_asset/mass_import.rst]: for this
and will also generate its variations

**Comment by juliensnz** [comment]
GTM

File: cookbook/product_asset/mass_import.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+If yes, the application will update the asset "demo_video" by importing the file "demo_video.wmv" as the new reference file of the asset (and will also generate the variations for the asset).
+If no asset exists with the code "demo_video", the PIM will create this new asset and will generate new variations.
```
Comment (rybus) [inline cookbook/product_asset/mass_import.rst]: and will generate its variations

File: cookbook/product_asset/mass_import.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+If you want to enrich these newly imported assets with other information (description, tags, etc.), you need to create an import profile for the job "Asset import in CSV".
```
Comment (rybus) [inline cookbook/product_asset/mass_import.rst]: (such as description, tags, etc.)

File: cookbook/product_asset/mass_import.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+The job accepts CSV a file in the same format as the one below (delimiters and escape characters can be configured in the interface):
+
+.. csv-table:: Assets import
+   :header-rows: 1
+   :file: asset.csv
+
+In the profile configuration of the import job, you can specify the path to the assets CSV file on the server, so that this file is imported when the job is executed.
+
+Then, the job can be can ran with:
+``php app/console akeneo:batch:job assets_mass_upload``
```
Comment (rybus) [inline cookbook/product_asset/mass_import.rst]: --env=prod

**Pull request merged**